packages/terra-heading/tests/nightwatch/HeadingVisuallyHidden.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Heading from '../../lib/Heading';
+
+const HeadingVisuallyHidden = () => (
```

Comment:
Add context for functional verification. The blank page looks like a broken example.